corehq/apps/app_manager/views/releases.py

```diff
@@ -124,14 +125,29 @@ def _get_batch(start_build=None, skip=None):
             app_es = app_es.is_released()
         if build_comment:
             app_es = app_es.build_comment(build_comment)
+
         results = app_es.exclude_source().run()
+        total_apps = results.total
         app_ids = results.doc_ids
         apps = get_docs(Application.get_db(), app_ids)
+
+        if build_comment:
```

Comment:
A comment here would be useful like
`User wants to search a specific version, entered as a build_comment`

how about renaming build_comment to search_string or query?

Reply:
02a26f6d46e